src/codegen/genVarsFile.ts

```diff
@@ -0,0 +1,46 @@
+import { Project } from "../document/2025/DocumentTypes";
+import { round, writeConst } from "./internals";
+
+export function genVarsFile(project: Project, packageName: string): string {
+  const expressions = project.variables.expressions;
+  const poses = project.variables.poses;
+  const out: string[] = [];
+  out.push(`package ${packageName};\n`);
+  out.push(`
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.measure.*;
+import static edu.wpi.first.units.Units.*;
+
+/**
+ * Generated file containing variables defined in choreo.
```

Comment:
```suggestion
 * Generated file containing variables defined in Choreo.
```
Please get the project name right in strings.